ci/run_envoy_docker.sh

```diff
@@ -55,6 +56,9 @@ fi
 ENVOY_DOCKER_BUILD_DIR="${ENVOY_DOCKER_BUILD_DIR//\\//}"
 mkdir -p "${ENVOY_DOCKER_BUILD_DIR}"
 
+DEFAULT_GOPROXY="https://proxy.golang.org,direct"
+[[ -z "${GOPROXY}" ]] && GOPROXY="${DEFAULT_GOPROXY}"
```

Comment:
Not sure we should default everyone to proxy.golang.org, we may just leave it blank?